config.go

```diff
@@ -517,7 +522,22 @@ func loadConfig() (*config, []string, error) {
 		fmt.Fprintln(os.Stderr, "prunetickets option is no longer necessary "+
 			"or used -- please update your config")
 	}
-
+	if len(cfg.TicketAddress) > 0 {
+		fmt.Fprintln(os.Stderr, "ticketaddress option is "+
```

Comment:
In addition to these warnings, it should also update the new options if they have not yet been set.  This keeps the behavior of old configs unmodified.